### DISCUSSION TITLE
Routing all remote clients onto internet via same gateway

### DISCUSSION THREAD

**Discussion by samgaw58** [dec0dOS/zero-ui#154] [Q&A] [answered]
Hi,

I hope someone can help me, I need to get all remote clients to route via our external IP address to use a certain service as it is linked to our external IP address.
So basically what I need is a user to connect to the Zerotier network hosted on our ZeroUi server, then when they use the internet it will appear as though they are using our office connection.

Is this possible using ZeroUi? I have use of Raspberry Pi's that could be configured in whatever way was needed.

Many thanks for this fantastic software.

**Comment by dec0dOS** [comment, marked as answer]
Hello, @samgaw58 

You should use 0.0.0.0/0 as a default route in ZeroUI for your server with IP forwarding enabled.

Check the following example
https://harivemula.com/2021/09/18/routing-all-traffic-through-home-with-zerotier-on-travel/

**Reply by samgaw58** [reply]
Many thanks, I'll give that a go today.
